soccer/src/soccer/radio/packet_convert.cpp

```diff
@@ -1,15 +1,40 @@
 #include "packet_convert.hpp"
 
-#include <rj_geometry/util.hpp>
 #include <rj_common/status.hpp>
 #include <rj_common/time.hpp>
+#include <rj_geometry/util.hpp>
 
+#include "control/motion_setpoint.hpp"
 #include "robot_intent.hpp"
 #include "robot_status.hpp"
-#include "motion/motion_setpoint.hpp"
 
-// TODO(#1583): Make this a real ROS parameter
-constexpr float kMaxKickSpeed = 7.0;
+// TODO(#1583): Make these ROS parameters and move them to a central location
+constexpr double kMaxKickSpeed = 7.5;
+constexpr double kMinKickSpeed = kMaxKickSpeed / 4;
+constexpr double kMaxChipSpeed = 3.0;
+constexpr double kMinChipSpeed = kMaxChipSpeed / 4;
+
+static uint8_t kicker_speed_to_strength(double kick_speed) {
+    return static_cast<uint8_t>(
+        std::min(1.0, (kick_speed - kMinKickSpeed) / (kMaxKickSpeed - kMinKickSpeed)) * kMaxKick);
+}
+
+static double kicker_strength_to_speed(uint8_t kick_strength) {
```

Comment:
We should probably by some mechanism or another make sure that there is unity between gameplay and this code as far as how fast we should expect a kick to come out, and ensuring that the range of values is correct.